src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_blob_copy_scenarios.py

```diff
@@ -106,3 +106,28 @@ def test_storage_blob_copy_same_account_sas(self, resource_group, storage_accoun
             actual_content = f.read()
 
         self.assertEqual(expect_content, actual_content)
+
+        # test source sas-token input starting with '?'
+        if not sas.startswith('?'):
+            sas = '?' + sas
+
+        target_container = self.create_container(account_info)
+        self.storage_cmd('storage blob copy start -b dst -c {} --source-blob src --source-sas {} --source-container {}',
```

Comment:
@troydai changed this test to use specify a sas token specifically for the source.